src/app/reporting/SynchronizedReportSchedulerImpl.h

```diff
@@ -59,7 +59,7 @@ class SynchronizedReportSchedulerImpl : public ReportSchedulerImpl, public Timer
     Timestamp mNextMinTimestamp = Milliseconds64(0);
 
     // Timestamp of the next report to be scheduled, only used for testing
```

Comment:
This comment needs to be fixed to explain what this thing really is and when it's used.

Reply:
Updated description.